Rakefile

```diff
@@ -34,3 +36,17 @@ end
 task :release do
   system 'npm publish'
 end
+
+desc 'Update the GitHub pages site'
+task :'gh-pages' do
```

Comment:
I'd like to move this out from the `Rakefile` to a multiline script in the `.travis.yml`